docs/components/stepper.md

```diff
@@ -0,0 +1,132 @@
+# Stepper
+
+**KStepper** - An ordered Stepper component
+
+<KStepper :steps="defaultItems" />
+
+```html
+<KStepper :steps="steps" />
+```
+
+## Props
+
+### steps
+
+An array of step objects. Each step object should have a required `label` property, and an optional `state` property.
+
+<div>
+  <KStepper :steps="[
+      { label: 'Step  a long long long long time ago', state: 'completed' },
+      { label: 'in a galaxy far far away', state: 'completed' },
+      { label: 'Kongponents were battling in space and', state: 'pending' },
+      { label: 'fighting robots and space monsters with lots of explosions' }
+    ]"
+  />
+</div>
+
+```html
+<KStepper :steps="[
+    { label: 'Step  a long long long long time ago', state: 'completed' },
+    { label: 'in a galaxy far far away', state: 'completed' },
+    { label: 'Kongponents were battling in space and', state: 'pending' },
+    { label: 'fighting robots and space monsters with lots of explosions' }
+  ]"
+/>
+```
+
+#### Properties
+
+- `label` (required) - the text displayed beneath the step
+- `state` - the state of the step controls the icon, we support: `completed`, `pending`, and `error`. If a 'state' is not provided, it will show the default grey icon.
```

Comment:
```suggestion
- `state` - the state of the step controls the icon, we support: `completed`, `pending`, and `error`. If a state property is not provided, it will show the default grey icon.
```